lib/tests/api.rs

```diff
@@ -86,6 +91,7 @@ mod api_integration {
 
 		#[test_log::test(tokio::test)]
 		async fn any_engine_can_connect() {
+			let _permit = FDB_PERMITS.acquire().await.unwrap();
```

Comment:
What about using explicitly using "`drop(permit);`". It is indeed not required, but we can be sure that no compilation optimisation would every release _permit too early? 

https://docs.rs/tokio/latest/tokio/sync/struct.Semaphore.html#prevent-tests-from-running-in-parallel

Reply:
Sounds like a good idea. Thanks!